src/components/validation/validators/component_spec/sources.rs

```diff
@@ -14,6 +14,7 @@ pub enum SourceMetricType {
     ReceivedBytesTotal,
     SentEventsTotal,
     SentEventBytesTotal,
+    EventsDropped,
```

Comment:
yeah this is a bit annoying because the actual metric name in `core` is `ComponentEventsDropped` , but we use `component_discarded _events_total` instead of `component_events_dropped_total` , whereas the others have consistency.